src/styles/style.css

```diff
@@ -0,0 +1,64 @@
+@font-face {
+  font-family: Avenir-Book;
+  src: url(/src/fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(/src/fonts/Avenir-Heavy.ttf);
+}
+
+.page__body {
+  margin-top: 0;
+}
+
+.form__search {
+  width: 100%;
+  margin-top: 20px;
```

Comment:
The `margin-top: 20px;` on `.form__search` applies to both search bars, but it does not ensure a 20px spacing between them. Consider using a `margin-bottom` on the first search bar or a `margin-top` on the second search bar to achieve the required spacing.